extensions/amp-image-viewer/0.1/amp-image-viewer.js

```diff
@@ -317,21 +331,23 @@ export class ImageViewer {
     // and then naturally upgrade to a higher quality image.
     return Services.timerFor(this.win).promise(1).then(() => {
       this.image_.setAttribute('src', src);
-      return this.loadPromise_(this.image_);
+      return Promise.resolve(this.image_);
```

Comment:
`Promise.resolve` is unnecessary here since returning the unwrapped value is equivalent inside a `then()` block.